contracts/treasury/ERC20Treasury.sol

```diff
@@ -46,4 +48,9 @@ contract ERC20Treasury is Context {
         //TODO contact the ERC20 contract and transfer
         //TODO emit transfer out event
     }
+
+    /// @notice Returns the API version
+    function getVersion() public pure override returns (string memory) {
```

Comment:
suggestion: as mentioned elsewhere, move this logic into `Version`

The idea being to have all the contract not require this common code, rather refactoring into their shared abstract contract 🙂